new/PRS13452Mamas.xml

```diff
@@ -53,6 +53,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">ማማስ፡</persName>
                       <persName xml:lang="gez" type="normalized" corresp="#n1">Māmās</persName>
+                      <persName xml:lang="gez" xml:id="n2" type="alt">ማማስ፡</persName>
```

Comment:
I would not restate the same name as "alt", there is no difference in spelling - I would put it all together

```
<persName xml:lang="gez" xml:id="n1">ማማስ፡</persName>
                      <persName xml:lang="gez" type="normalized" corresp="#n1">Māmās</persName>
<persName xml:lang="gez" corresp="#n1" type="alt">Māmmās</persName>
```

Reply:
Ah,ok.